travisify.sh

```diff
@@ -166,7 +166,7 @@ EOL
 				'#'*) continue;;
 			esac
 			info "Encrypting ${p%%=*}"
-			echo yes | $EXEC travis encrypt "$p" --add env.global
+			echo yes | $EXEC travis encrypt "$p" --add env.global --repo $repoSlug
```

Comment:
With this change, I think the `echo yes |` is no longer necessary. I'll remove it in the following commit.